src/Camille.RiotGames/src/Enums/RegionConfig.cs

```diff
@@ -0,0 +1,18 @@
+﻿// MobaHinted Copyright (C) 2024 Ethan Henderson <ethan@zbee.codes>
+// Licensed under GPLv3 - Refer to the LICENSE file for the complete text
+
```

Comment:
Presumably this was autogenerated - definitely need to remove this and also double check that it's ok to release your PR under the existing Apache2+ MIT license?